src/components/HomepageFeatures/index.tsx

```diff
@@ -63,13 +63,13 @@ const FeatureList: FeatureItem[] = [
     ),
   },
   {
-    title: 'Use WebAssembly(WASM)',
+    title: 'Use WASM with Swanky',
```

Comment:
Should we rename this `Use WASM smart-contracts` (as opposed to `use EVM`) and have a dedicated separated section for `Swanky` ?

Reply:
That is open for discussion - The whole PR started after we identified the need to bring more visibility to Swanky on the front page.

We could do it like that, but where would the Swanky link point to then? Seems redundant if it would point to the (more or less) same place in the documents?